tests/chains/NEAR/TWAnySignerTests.cpp

```diff
@@ -127,4 +127,39 @@ TEST(TWAnySignerNEAR, SignUnstakeMainnetReplication) {
     ASSERT_EQ(Base64::encode(data(output.signed_transaction())), "QAAAAGI4ZDVkZjI1MDQ3ODQxMzY1MDA4ZjMwZmI2YjMwZGQ4MjBlOWE4NGQ4NjlmMDU2MjNkMTE0ZTk2ODMxZjJmYmYAzgCT6NK76nb1mB7pToefgkGUHfUe5BKvvr3gW/nq+MgGuu1Mq0YAABEAAABhdmFkby5wb29sdjEubmVhcq0YnhRlt+TTtagkoy0qKn56zAfGhE+jkTJW6PR5k5r8AQAAAAILAAAAdW5zdGFrZV9hbGwCAAAAe30A0JjUr3EAAAAAAAAAAAAAAAAAAAAAAAAABaFP0EkfJU3VQZ4QAiTwq9ebWDJ7jx7TxbA+VGH4hwKX3gWnmDHVve+LK7/UbbffjF/y8vn0KrPxdh3ONAG0Ag==");
 }
 
+/// Implements NEP-141:
+/// https://nomicon.io/Standards/Tokens/FungibleToken/Core
+///
+/// Successfully broadcasted tx
+TEST(TWAnySignerNEAR, SignTokenTransfer) {
+    auto privateKey = parse_hex("77006e227658c18da47546413926a26b839204b1b19e807c4a13d994d661c72e");
+
+    auto blockHash = Base58::decode("2dQBYs8XjprLLgtH7eVsJ3e58A5QuRcbuqFisSk9fFWQ");
+
+    // Deposit should be 1 yocto NEAR for security purposes.
+    auto amount = parse_hex("01000000000000000000000000000000");
+
+    auto transfer_args = R"r({"amount":"100000000000000000","receiver_id":"c6d5e3e8f328436f595856a598239b691d3d136b24c05a4614f9e9716edc14fe"})r";
```

Comment:
Can we add a `TokenTransfer` message to `NEAR.proto`? so user don't need to prepare the json and the function call by themselves

Reply:
Good idea, will do

Reply:
I had to add a new action type called `TokenTransfer`, but its standard ID is the same as `FunctionCall`'s, so I had to do the following trick:
https://github.com/trustwallet/wallet-core/blob/e4033bdcc8bd16aa8c5a2954439e6f1093231d16/src/NEAR/Serialization.cpp#L131-L136